app/scripts/modules/core/application/modal/pageApplicationOwner.modal.controller.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.pageApplicationOwner.modal.controller', [
+  require('../service/applications.write.service.js'),
+  require('../../../core/task/monitor/taskMonitorService.js'),
+])
+  .controller('PageApplicationOwner', function ($scope, $uibModalInstance,
+                                                taskMonitorService, applicationWriter, application) {
+    var vm = this;
+    $scope.application = application;
+    $scope.command = {
+    };
+
+    vm.submit = function () {
+      var taskMonitorConfig = {
+        application: application,
+        title: 'Paging ' + application.name + ' owner',
+        modalInstance: $uibModalInstance
+      };
+
+      var submitMethod = function () {
+        return applicationWriter.pageApplicationOwner(
+          $scope.application, $scope.command.reason
+        );
+      };
+
+      $scope.taskMonitor = taskMonitorService.buildTaskMonitor(taskMonitorConfig);
+      $scope.taskMonitor.submit(submitMethod);
+    };
+  });
```

Comment:
Avoid using `$scope` on new code - we are trying to keep everything on the controller, so just change a few things:
```javascript
     this.application = application;
     this.command = {};
 
     this.submit = () => {
       var taskMonitorConfig = {
         application: application,
         title: 'Paging ' + application.name + ' owner',
         modalInstance: $uibModalInstance
       };
 
       var submitMethod = () => {
         return applicationWriter.pageApplicationOwner(application, this.command.reason);
       };
 
       this.taskMonitor = taskMonitorService.buildTaskMonitor(taskMonitorConfig);
       this.taskMonitor.submit(submitMethod);
     };
```
You'll need to tweak those references in the template, too - just change `application` to `ctrl.application`, `command` to `ctrl.command`, `taskMonitor` to `ctrl.taskMonitor`, etc.